src/game/CowGame.tsx

```diff
@@ -28,13 +29,23 @@ export function CowGame() {
 
   // Hide box when pressing Space
   const hideMessageBox = () => setShowMessageBox(false);
-  useKeyPress(["Space", "ArrowUp"], hideMessageBox);
+
+  // Desktop/non-touch device: Key binding to hide message box
+  !!IS_MOBILE && useKeyPress(["Space", "ArrowUp"], hideMessageBox);
+
+  // Mobile/touch device: On touch start hide message box
+  const messageBox = document.getElementById("main-cowgame");
```

Comment:
Saving the parent wrapping container as an element, so it doesn't matter where you tap on screen to start (for touch devices).